cypress.json

```diff
@@ -1,5 +1,7 @@
 {
   "baseUrl": "http://localhost:3000",
   "chromeWebSecurity": false,
-  "video": false
+  "video": false,
+  "integrationFolder": "cypress/",
```

Comment:
Defaults to `cypress/integration`, thought it'd be nice to have the e2e tests in a separate `cypress/endToEnd` directory - let me know what you think